hedera-node/hedera-addressbook-service-impl/src/main/java/com/hedera/node/app/service/addressbook/impl/validators/AddressBookValidator.java

```diff
@@ -151,31 +150,20 @@ public void validateAdminKey(@Nullable Key key) throws PreCheckException {
      */
     public void validateAccountId(@Nullable AccountID accountId) throws PreCheckException {
         validateAccountID(accountId, INVALID_NODE_ACCOUNT_ID);
-        validateFalsePreCheck(!accountId.hasAccountNum() && accountId.hasAlias(), INVALID_NODE_ACCOUNT_ID);
+        validateFalsePreCheck(
+                !requireNonNull(accountId).hasAccountNum() && accountId.hasAlias(), INVALID_NODE_ACCOUNT_ID);
```

Comment:
line 152, validateAccountID() checked accountId is null, we don't need to add requireNonNull check here.

Reply:
I just like to get rid of the IntelliJ warnings. 😳